helpers/pluck.js

```diff
@@ -1,8 +1,21 @@
 'use strict';
 
+const helpers = require('handlebars-helpers');
+
 const factory = () => {
     return function(collection, path) {
-        return collection.map(item => item[path]);
+        if (collection === null || typeof collection === 'undefined' || typeof collection !== 'object') {
+            return '';
+        }
+
+        let res = [];
+        for (let i = 0; i < collection.length; i++) {
```

Comment:
Can you iterate through objects like this? I've never done it that way. Can you add a test case?